integrations/docker/images/chip-build-remote-builder/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM google/cloud-sdk
```

Comment:
Do we want this to be picked up by build-all and pushed into dockerhub?

https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/build-all.sh#L24

will pick up all dockerfiles unless we add some exclusion

Reply:
It doesn't hurt if this docker get  picked up by build-all and pushed into dockerhub like any other dockers, I uploaded this docker to gcr.io, we can choose to use this docker from either places.

Will check how to add exclusion if you think this extra backup is not necessary .

Reply:
From my test, it looks gcr.io is faster than  dockerhub, adding exclusion to prevent picking up to dockerhub